api/src/modules/geo-regions/geo-region.repository.ts

```diff
@@ -30,6 +30,8 @@ export class GeoRegionRepository extends Repository<GeoRegion> {
       .createQueryBuilder()
       .select(`hashtext(concat($3::text, points.radius))`)
       .addSelect(`points.radius`)
+      .addSelect(`array(SELECT h3_polyfill(points.radius,6))`)
```

Comment:
I know that this full query is out of this PRs scope, but have a look how the (currently) 2 possible geo-regions are saved, query-wise. 

I think both of them should follow the same approach with the query, therefore improving readability and improving our overall QoL

Do you think it's worth refactoring one of them to both follow the same approach?